NEOSPlus/Components/Transform/Drivers/MultiValueArithmeticDriver.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Linq;
+using BaseX;
+using FrooxEngine;
+
```

Comment:
There needs to be a namespace defined here,
```cs
namespace FrooxEngine;
```
should do.

(Linka please fix the namespaces in your code it's missing in a couple of places and dropping stuff to the global namespace is bad)